src/python/grpcio/grpc/_adapter/_c/types/channel.c

```diff
@@ -164,7 +164,7 @@ PyObject *pygrpc_Channel_watch_connectivity_state(
   int last_observed_state;
   CompletionQueue *completion_queue;
   char *keywords[] = {"last_observed_state", "deadline",
-                      "completion_queue", "tag"};
+                      "completion_queue", "tag", NULL};
```

Comment:
Okay, I am able to reproduce it, and it seems to be nondeterministic in my primary development environment and deterministically hanging in my secondary development environment. What do you make of "RuntimeError: More keyword list entries (9) than format specifiers (4)" and its relation to the addition of NULL at the end of this array?